Criterion/Report.hs

```diff
@@ -129,11 +129,6 @@ escapeJSON '\x2029' = "\\u2029" -- paragraph separator
 escapeJSON '&'      = "\\u0026" -- avoid HTML entities
 escapeJSON '+'      = "\\u002b" -- + can be used in UTF-7 escape sequences
 escapeJSON '\0'     = "\\u0000" -- make null characters explicit
-escapeJSON '\n'     = "\\u000a" -- for good measure also escape newlines
```

Comment:
The Haddocks for `escapeJSON` above mention that `\n`, `\\` and friends are escaped, which is no longer true. Please update the Haddocks, and perhaps mention why `\\` _isn't_ escaped.